components/ai_chat/core/engine/engine_consumer_llama.h

```diff
@@ -61,6 +61,8 @@ class EngineConsumerLlamaRemote : public EngineConsumer {
 
   std::unique_ptr<RemoteCompletionClient> api_ = nullptr;
 
+  int max_page_content_length_ = 0;
```

Comment:
We could consider having the whole Model on this class (or even the super class), but this is fine for now. It doesn't need the rest of the data for now.